receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_scraper.go

```diff
@@ -59,29 +59,30 @@ func (s *scraper) Close(_ context.Context) error {
 func (s *scraper) ScrapeMetrics(_ context.Context) (pdata.MetricSlice, error) {
 	metrics := pdata.NewMetricSlice()
 
+	now := pdata.TimestampUnixNano(uint64(time.Now().UnixNano()))
```

Comment:
Please add an `internal.TimeToUnixNano` similar to `internal.UnixNanoToTime` and use that